internal/panda_controller/docker_grpc_agent.go

```diff
@@ -192,3 +193,32 @@ func (pa *dockerGrpcPandaAgent) stopContainer(ctx context.Context) error {
 
 	return nil
 }
+
+func copyFileHelper(file_path string, shared_dir string) (int64, error) {
+
+	sourceFileStat, err := os.Stat(file_path)
+	if err != nil {
+		return 0, err
+	}
+
+	if !sourceFileStat.Mode().IsRegular() {
+		return 0, fmt.Errorf("%s is not a regular file", file_path)
+	}
+
+	source, err := os.Open(file_path)
+	if err != nil {
+		return 0, err
+	}
+	defer source.Close()
+
+	destination, err := os.Create(shared_dir + "/system_image.qcow2")
```

Comment:
If you ever want more than one qcow in the shared directory this would need to not be hardcoded. I believe each docker instance has its own shared directory, but something to remember if an issue comes up. This also blocks the ability to add more than one file to the shared directory using this function, and it has to be a qcow. While it works for the situation, if you ever want to use this function again, you need to consider this.

Reply:
I added in a parameter for the file name for better reuse of the method. In terms of the hard coded value, that will be changed once we take care of getting the rest of the image information such as arch and everything and we can update that there so its not specific